src/Domain/EmbeddingMaps/AffineMap.hpp

```diff
@@ -14,19 +14,27 @@
 
 namespace EmbeddingMaps {
```

Comment:
Constructors (and any other non-virtual functions) can't be called through base class pointers.  We can debate whether they are _worth_ inlining, but the opportunity to do so is definitely there.